docs/Rules/UseSingularNouns.md

```diff
@@ -13,6 +13,27 @@ title: UseSingularNouns
 
```

Comment:
Is there a need to change `ms.custom` version to `v1.22.0`, as well as the date?